docs/enterprise-edition-guide/README.md

```diff
@@ -14,11 +14,11 @@ The Enterprise Edition builds on top of the free Community Edition. An upgrade i
 
 | Popular Topics                                               | Description                                                  |
 | ------------------------------------------------------------ | :----------------------------------------------------------- |
-| [Enterprise Edition trial](#create-an-enterprise-edition-trial) | Can I test the Enterprise Edition for free?                  |
-| [Activate the Enterprise Edition](#activate-the-enterprise-edition) | How can I upgrade my Community Edition to an Enterprise Edition? |
-| [Installation support](#installation-support)                 | How can I get installation support?                          |
+| [Enterprise Edition trial](/enterprise-trial)                | Can I test the Enterprise Edition for free?                  |
```

Comment:
I would also change the wording here and in the following section to "Enterprise on-premises" trial.